test/mobile/test_upgrader_bytecode_table_example.cpp

```diff
@@ -279,6 +287,99 @@ const std::vector<ByteCodeFunctionWithOperator>& getUpgraderBytecodeList() {
                                    OperatorString({"aten::div", "out_mode", 4}),
                            }), // operators list
                    }),
+                   ByteCodeFunctionWithOperator({
```

Comment:
Is this part of the regular process when writing an upgrader?

Reply:
Due to limitations on how XPLAT buck works, we decided that it is impossible to add bytecode generation process as a build process, therefore we decided to go with the approach where we check in bytecode cpp file. Users who are introducing new BC changes are required to regenerate this file if there is any modification. But this is documented in a google doc and will be easy to do. @cccclai

Reply:
Do you have a pointer to the Google Doc?

Reply:
This process can be done by running
```
python pytorch/tools/codegen/operator_versions/gen_mobile_upgraders.py
```
as it describes in [the draft instruction](https://github.com/pytorch/pytorch/blob/1df4eca6d7ab0885562cafb5b69f1ce35d448396/test/jit/fixtures_srcs/generate_models.py#L24-L82). We are also looking into consolidating a few steps so it's easier to use. Is there any concern from the instruction?

Reply:
Running a script seems OK but it'd be nice if OSS contributors had access to all the tools for this process, too.

Also one of the script's locations should probably be edited from: 
```
python /Users/chenlai/pytorch/test/jit/fixtures_src/generate_models.py
```

We should think about including the relevant pointers in a Wiki page, too, for easier editing.